src/openrct2-ui/ride/Construction.cpp

```diff
@@ -38,6 +38,7 @@ namespace OpenRCT2
         TrackElemType::Down25Brakes,
         TrackElemType::BlockBrakes,
         TrackElemType::DiagBlockBrakes,
+        TrackElemType::DiagDown25Brakes,
```

Comment:
Needs to be moved up a bit, so that all brakes come first, following by all block brakes.